doc/hand_eye_calibration/hand_eye_calibration_tutorial.rst

```diff
@@ -0,0 +1,142 @@
+"Object frame" is the frame defined by the calibration target, which is called "handeye_target" by default, and the
+remaining frames are the robot links rigidly attached to the camera (the "End-effector frame") and the calibration
+target (the "Robot base frame"; although the target might not be literally "rigidly attached", it doesn't move in this
+frame).
+
+.. image:: images/context_tab.png
+
+The FOV section controls the rendering of the camera's field of view in RViz. To see the FOV, add a "MarkerArray"
+display, and set it to listen to the "/rviz_visual_tools" topic. (It may not appear immediately.)
+
+Finally, it is not necessary to set an initial guess for the camera pose, but it is worth noting that once a calibration has been
+calculated, these fields will be updated with the new calibration.
+
+Collect Dataset
+---------------
+Next, we will capture a calibration dataset. The robot kinematics provide the end-effector's pose in the robot base
+frame, and the calibration target's pose in the camera frame can be estimated, as mentioned above. If the target's pose
+in the robot base frame were known accurately, only a single observation of the camera-target transform would be
+necessary to recover the camera's pose in the end-effector frame. The direct camera-to-end-effector transform is
+equivalent to the composite camera-to-target-to-base-link-to-end-effector transform. A better option, however, is to
+combine the information from several poses to eliminate the target pose in the base frame from the equation, as
+described in `this paper by Kostas Daniilidis <https://scholar.google.com/scholar?cluster=11338617350721919587>`_.
+
+Our dataset, then, comprises several pairs of poses: the end-effector's pose in the robot base frame paired with the
+calibration target's pose in the camera frame. Once five such pose pairs have been collected, the calibration can be
+calculated.
+
+The "Calibrate" tab provides the tools to collect the dataset and calculate and export the calibration. At this point,
+it is also helpful to add an image panel to the RViz display to see the target detection in the camera view, which is
+published on ``/handeye_calibration/target_detection``.
+
+.. image:: images/calibrate_tab.png
+
+On the "Calibrate" tab, you can select which calibration solver to use in the "AX=XB Solver" drop-down. The Daniilidis
+solver (from the paper referenced, above) is the default and is a good choice in most situations. The "Planning Group"
+is the joint group that will be recorded, so should be set to the appropriate group for the arm (in the
+``panda_moveit_config`` package, the ``panda_arm`` group should be used).
+
+When the target is visible in the arm camera, and the axis is rendered on the target in the target detection image, you
+are ready to take your first calibration sample. Click the "Take sample" button in the "Manual calibration" section, and
```

Comment:
```suggestion
are ready to take your first calibration sample (= one pose pair). Click the "Take sample" button in the "Manual calibration" section, and
```

Reply:
Good call. I also update the prior paragraph to call a pose pair a "sample" from the beginning.